api/src/main/java/graphql/nadel/schema/NadelDirectives.java

```diff
@@ -56,7 +56,7 @@
     static {
         DYNAMIC_SERVICE_DIRECTIVE_DEFINITION = DirectiveDefinition.newDirectiveDefinition()
                 .name("dynamicServiceResolution")
-                .directiveLocation(newDirectiveLocation().name(FIELD_DEFINITION.name()).build())
+                .directiveLocation(newDirectiveLocation().name(INTERFACE.name()).build())
```

Comment:
Hang on - this means it can only be on interfaces (and not fields) ??

Down later code assumes its on fields

Reply:
You're right!
The directive should be allowed to be on fields. 
I'll change this back.